starters/vue3-xstate-css/src/stories/ButtonComponent.stories.js

```diff
@@ -0,0 +1,19 @@
+import ButtonComponent from '../components/ButtonComponent.vue';
```

Comment:
Is this the default story that automatically comes with storybook?
If so, I think we should remove it because the spec mentions removing all default stories.

Reply:
Nope this is a specific one I made for this kit. :)